app.js

```diff
@@ -5,6 +5,7 @@ var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 var config = require('./config/config');
 require('./config/globals')();
+require("dotenv").config();
```

Comment:
@priyanka-TL - Please config.json as already done in case of dhiti service for now.